app/script/entity/message/MessageTimerUpdateMessage.js

```diff
@@ -0,0 +1,42 @@
+/*
+ * Wire
+ * Copyright (C) 2018 Wire Swiss GmbH
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see http://www.gnu.org/licenses/.
+ *
+ */
+
+'use strict';
+
+window.z = window.z || {};
+window.z.entity = z.entity || {};
+
+z.entity.MessageTimerUpdateMessage = class MessageTimerUpdateMessage extends z.entity.SystemMessage {
+  constructor(messageTimer) {
+    super();
+
+    this.type = z.event.Backend.CONVERSATION.MESSAGE_TIMER_UPDATE;
+    this.system_message_type = z.message.SystemMessageType.CONVERSATION_MESSAGE_TIMER_UPDATE;
+    this.message_timer = messageTimer;
+
+    this.caption = ko.pureComputed(() => {
+      if (this.message_timer === null) {
```

Comment:
Can we safely go with a null here or should we just check for falsy instead incase we have undefined or 0?